client/src/plus/notifications/index.tsx

```diff
@@ -1,148 +1,104 @@
+    offset,
+    selectedTerms,
+    selectedFilter,
+    selectedSort,
+    currentTab
+  );
 
-  if (location.pathname === starredUrl) {
-    apiUrl += "&starred=true";
-    pageTitle = "My Starred Pages";
-  }
   useEffect(() => {
-    const clearNotificationsUrl =
-      "/api/v1/plus/notifications/all/mark-as-read/";
-    const formData = new FormData();
-    formData.append("csrfmiddlewaretoken", getCookie("csrftoken") || "");
-
-    // if the user clicks a hard link, we set notifications as read using a sendBeacon request
-    const markNotificationsAsRead = () => {
-      if (document.visibilityState === "hidden") {
-        navigator.sendBeacon(clearNotificationsUrl, formData);
-      }
-    };
-    document.addEventListener("visibilitychange", markNotificationsAsRead);
-
-    return () => {
-      // if the user clicks a react-router Link, we remove the sendBeacon handler
-      // and send a fetch request to mark notifications as read
-      document.removeEventListener("visibilitychange", markNotificationsAsRead);
-      fetch(clearNotificationsUrl, {
-        body: formData,
-        method: "POST",
-      }).then(async () => {
-        await mutate(apiUrl);
-        await mutate(HEADER_NOTIFICATIONS_MENU_API_URL);
-      });
-    };
-  }, [apiUrl]);
-
-  let watchingApiUrl = `/api/v1/plus/watched/?q=${encodeURIComponent(
-    selectedTerms
-  )}`;
+    let unread;
+    document.title = TAB_INFO.get(currentTab)?.pageTitle || "MDN Plus";
+    if (data && data.items) {
+      unread = data.items.filter((v) => v.read === false).length;
+    }
+    if (!!unread) {
+      document.title = document.title + ` (${unread})`;
+    }
+  }, [data, currentTab]);
 
-  const watching = location.pathname === watchingUrl;
-
-  const tabs = [
-    {
-      label: "All notifications",
-      path: `/${locale}/plus/notifications`,
-    },
-    {
-      label: "Starred",
-      path: starredUrl,
-    },
-    {
-      label: "Watch list",
-      path: watchingUrl,
-    },
-  ];
-
-  const filters = [
-    {
-      label: "Content updates",
-      param: "filterType=content",
-    },
-    {
-      label: "Browser compatibility",
-      param: "filterType=compat",
-    },
-  ];
+  useEffect(() => {
+    setSelectedSort("");
+    setSelectedTerms("");
+    setSelectedFilter("");
+    setOffset(0);
+  }, [currentTab, setSelectedSort, setSelectedTerms, setSelectedFilter]);
 
-  const sorts = [
-    {
-      label: "Date",
-      param: "sort=date",
-    },
-    {
-      label: "Title",
-      param: "sort=title",
-    },
-  ];
+  useEffect(() => {
+    setOffset(0);
+  }, [selectedTerms, selectedFilter, selectedSort]);
 
   return (
     <>
       <header className="plus-header">
         <Container>
           <h1>Notifications</h1>
         </Container>
-        <Tabs tabs={tabs} />
+        <Tabs
+          tabs={[...TAB_INFO.values()].map((val) => {
+            return { ...val, path: `/${locale}${val.path}` };
+          })}
+        />
```

Comment:
Does this just render the tab headings or the headings and content? If the latter, we probably do not want to put that into a `header` element.